### ISSUE TITLE
Channels first

### ISSUE THREAD

**Issue by APJansen** [UNSAT3D/unsat#10]
I had added a channels axis of size 1 (since we have greyscale images) at the end, as is the convention in Keras, but pytorch requires it to be at the front (so shapes (batch_size, channels, height, width) for 2D images.

Since where you want it may also depend on whether you're using 2D or 3D inputs, I'll just remove it entirely, and have the data loaders add it where needed. I've already updated the script and readme in the data repo.
I'll update the data on snellius as well, and the code here.
